packages/fast-components/src/card/fixtures/card.html

```diff
@@ -1,4 +1,4 @@
-<fast-design-system-provider>
+<fast-design-system-provider use-defaults>
```

Comment:
"use-defaults" seems to be the default way to consume this. That is, it's added here a number of times, though places it wasn't added obviously wouldn't show as a diff. Should / could this be the default behavior?

Reply:
Yea I just had a thought about this while writing the documentation. We only want to provide custom property values when the value is being explicitly set on the element because otherwise we would clobber the cascade and not inherit upstream values. `use-defaults` is essentially a short-hand to apply all the values. We could *also* apply the defaults if there is no upstream provider that was found - this would handle the root design system case in a way that doesn't require setting all of the attributes.

Reply:
Took a stab at this but it isn't as straight forward as I thought. Its a bit tricky to determine exactly when design system should use default values so I think we should just keep this opt-in for at the time being. This component is going to churn quite a bit I expect so I think this is something we can address as it hardens.

Reply:
I'd say if we're already checking for an upstream provider we should apply the defaults.

Reply:
Again - its not very straight forward. We can take as a follow up.

Reply:
https://github.com/microsoft/fast-dna/issues/2886